corehq/apps/importer/util.py

```diff
@@ -346,15 +347,37 @@ def get_spreadsheet(download_ref, column_headers=True):
         return None
     return ExcelFile(download_ref.get_filename(), column_headers)
 
+
+def is_location_group(owner_id, domain):
+    """
+    Return yes if the specified owner_id is one of the
+    faked location groups.
+    """
+    group_ids = [
```

Comment:
For posterity: Couldn't just replace it as this check also handles verifying the group is for a location in this domain, but I refactored.